### PR TITLE
Events: Show 'No Search Results' Text

### LINKED ISSUE #616: Add 'No Search Results' Text when empty object returned
Currently, no text is shown if the filter of the past events return empty object. Also, in case no future events are there, a text should be displayed.

### PR DESCRIPTION
**Pull request by akshatnitd**
Shows 'No Search result' text for past and future events

Fixes #616

### CODE REVIEW

**Comment by akshatnitd** [comment]
@alanmoo Added the 'No Search result' text for past events filtering and future events, similar to the projects page. Please have a look :)

File: app/react/pages/events.jsx [hunk 1]
```diff
@@ -113,7 +119,8 @@ export default React.createClass({
         </div>
         <div className="container-dynamic">
           <EventList cardClass="col-sm-6 col-md-4 archive" pictures={false} events={this.state.past.events} />
-          <div className="text-xs-center">
+          <div className="lead text-xs-center mb-2">
+            {this.state.past.empty && <p>Sorry, no past events found in this category.</p>}
```
Comment (alanmoo) [inline app/react/pages/events.jsx]: @auremoser copy?

Comment (auremoser) [inline app/react/pages/events.jsx]: Good good.

File: app/react/pages/events.jsx [hunk 2]
```diff
@@ -103,7 +108,8 @@ export default React.createClass({
         </div>
         <div className="container-dynamic">
           <EventList events={this.state.future.events} />
-          <div className="text-xs-center">
+          <div className="lead text-xs-center mb-2">
+            {this.state.future.empty && <p>Sorry, no events found in the near future.</p>}
```
Comment (auremoser) [inline app/react/pages/events.jsx]: Sounds good.

**Pull request merged**